exonum-java-binding-time-service/src/main/java/com/exonum/binding/time/serialization/UtcZonedDateTimeSerializer.java

```diff
@@ -15,29 +15,32 @@
  *
  */
 
-package com.exonum.binding.common.serialization;
+package com.exonum.binding.time.serialization;
```

Comment:
Can we put it in `com.exonum.binding.time` and make package-private?

Reply:
Or will it be useful in tests that want to (though this is not recommended, but nothing better is available at the moment) put something in TimeSchema?

Reply:
@MakarovS Have you researched that?

Reply:
I couldn't think of any cases when someone wants to put something in TimeSchema that we want to support right now. I'd leave it private and change it in the future in case such a need ever appears (and it's very possible that we'd move to protobuf earlier and would change it anyway).